service/relayer/migrations/20260626120000_relay_operation_key.up.sql

```
@@ -0,0 +1,49 @@
+-- can be null until the gateway returns it.
+ALTER TABLE "transaction"
+DROP CONSTRAINT IF EXISTS pk__transaction;
+
+ALTER TABLE "transaction"
+ADD CONSTRAINT pk__transaction PRIMARY KEY (operation_key);
+
+ALTER TABLE "transaction"
+ALTER COLUMN transaction_hash DROP NOT NULL;
+
+ALTER TABLE account
+ADD CONSTRAINT fk__account__transaction FOREIGN KEY (pending_operation_key) REFERENCES "transaction" (operation_key);
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)